core/iwasm/fast-jit/fe/jit_emit_conversion.c

```diff
@@ -64,6 +64,10 @@ jit_compile_op_i32_trunc_f32(JitCompContext *cc, bool sign, bool saturating)
         }
         *(jit_insn_opndv(insn, 2)) = value;
 
+#if defined(BUILD_TARGET_X86_64) || defined(BUILD_TARGET_AMD_64)
+        jit_lock_regs_in_insn(cc, insn, nan_ret);
```

Comment:
Should check the return value? it may return false

Reply:
a little struggle about it. 
The possibility of failure is about creating instructions failed. If going to check the return value here, we should also cover all `GEN_INSN` and other similar situations.

Maybe changing the return value of `jit_lock_reg_in_insn` is another way. what do u think?

Reply:
How about set the return value to void, and in the jit_lock_reg_in_insn: jit_set_last_error when gen_MOV insn failed? So the compiler can detect the error after this pass finishes.

Reply:
Or not change the return type, but add jit_set_last_error?